examples/with-apollo/src/app/page.tsx

```diff
@@ -1,9 +1,21 @@
-import { ApolloClientProvider, RepoList } from "@/client-components";
+import { FiveRockets } from "@/components/FiveRocketsClient";
+import { LatestMissionName } from "@/components/LatestMissionName";
+import { Suspense } from "react";
 
 export default async function Home() {
   return (
-    <ApolloClientProvider>
-      <RepoList />
-    </ApolloClientProvider>
+    <>
+      <article>
+        <h2>
+          Latest mission: <LatestMissionName />
+        </h2>
+      </article>
+      <article>
+        <h2>Five Rockets:</h2>
+        <Suspense fallback={<div>loading...</div>}>
```

Comment:
[nit] Should this use the `loading` component `'./loading'`? (I'm also unfamiliar with Next so perhaps this is an ignorant question).

Reply:
No, `loading.ts` is a Next.js concept - it's a full-page suspense boundary wrapping the whole page on the router level.

I kinda want to show here that it can be handled both ways.